### ISSUE TITLE
Option to not have people marked "Sociable" prioritize above "Online" users

### ISSUE THREAD

**Issue by LeCloutPanda** [Yellow-Dog-Man/Resonite-Issues#2197]
### Is your feature request related to a problem? Please describe.

Having users marked as "Sociable" appear at the top of the list offsetting everyone else, I understand it is meant for "Hey ping me" etc but I think it would be good to let us have those users mixed together with "Online" users.

### Describe the solution you'd like

A toggle named "Prioritize Sociable users to top of list" and when it's off it just mixes the users in with the "Online" users

### Describe alternatives you've considered

Can't do anything in game without doing some stupid hacky shit.

### Additional Context

As it is just a status people will leave it on and not use it as intended so having them not be at the top of the list makes things easier to filter through people.

### Requesters

[LeCloutPanda](https://github.com/LeCloutPanda) (Discord)
People from the discussion [thread](https://discord.com/channels/1040316820650991766/1246024940256366632) and github issue https://github.com/Yellow-Dog-Man/Resonite-Issues/discussions/2186

**Comment by ExoTheWicker** [comment]
A more complex idea I have is have a priortioty setting (simular to the audio settings) where you can assign priorities

Example:

(Headless)
(Sociable)
(Online)
(Busy)
(away)
(offline)

if someone want to have to that way

**Comment by TisFoolish** [comment]
Honestly I'd love to put headlesses above everything else

**Comment by shiftyscales** [comment]
This is functionally a duplicate of #41, @LeCloutPanda. The existing contacts UI currently uses a hardcoded priority list to sort contacts.

I understand the need, and desire for this functionality- I'd suggest users to thumb up #41 and the issue it is currently blocked by- the contacts UI rework #1116 to signal interest in its development.

With that said- I am closing this as a duplicate- thank you for your feedback.
